src/components/Footer/NavigationButton.tsx

```diff
@@ -1,16 +1,16 @@
-import react from 'react';
+import React from 'react';
 import { useNavigate } from 'react-router-dom';
 import {
   NAVIGATION_BUTTON_STYLE,
   NAVIGATION_BUTTON_IMAGE_STYLE,
   NAVIGATION_BUTTON_TEXT_STYLE,
-} from '../../constants/styles';
+} from '@/constants/styles';
 
-import type { NavigationButtonProps } from '../../types/navigationBar/navigationBarTypes';
+import type { NavigationButtonProps } from '@/types/navigationBar/navigationBarTypes';
 
-export function NavigationButton(props: NavigationButtonProps) {
+export const NavigationButton = (props: NavigationButtonProps) => {
   const { navigationURL, alt, icon, hoverIcon, labelText } = props;
-  const [buttonIcon, setButtonIcon] = react.useState(icon);
+  const [buttonIcon, setButtonIcon] = React.useState(icon);
```

Comment:
보통 react 안에 있는 기본 구성요소(useState, useEffect 등...)는 바로 요소를 불러오지 않나요?
```suggestion
  const [buttonIcon, setButtonIcon] =useState(icon);
```
import 하실 때
```
import { useEffect, useState } from "react";
```
이런식으로 하는게 좋지 않을까요?

Reply:
반영되었습니다